pkg/agent/core/ngt/handler/grpc/handler_test.go

```diff
@@ -4302,6 +4302,813 @@ func Test_server_MultiInsert(t *testing.T) {
 				},
 			}
 		}(),
+
+		func() test {
+			req := genF32Req(vector.Gaussian, 100, f32VecDim)
```

Comment:
Do you have a plan using https://github.com/vdaas/vald/pull/1645/files#diff-059e77835fd022fce77c6202e61ad5a1148830b650e633e31fbee8ab1f824431R3746 instead of it?

Reply:
fixed